UPGRADE-1.2.md

```diff
@@ -5,3 +5,11 @@ Core\Test:
     * For the context to work it does require BehatchBridgeExtension enabled in Behat configuration
 * The signature of constructor `ApiAuthTokenContext` has changed
     * For the context to work it does require BehatchBridgeExtension enabled in Behat configuration
+
+ExporterFile:
+* The signature of constructor `StartProcessCommandHandler::__construct(ExportRepositoryInterface $repository, TempFileStorage $storage, ExportProductBuilder $productBuilder, ExportTemplateElementBuilder $templateElementBuilder, ExportAttributeBuilder $attributeBuilder, ExportOptionBuilder $optionBuilder, ExportCategoryBuilder $categoryBuilder, ExportTemplateBuilder $templateBuilder)`
```

Comment:
Just stick with the information that the constructor has changed(as in UPGRADE 1.1.md)